--- pkg/assets/bindata.go	2023-04-11 12:33:12.134678061 -0300
+++ pkg/assets/bindata.go-ok	2023-04-11 12:33:07.950684962 -0300
@@ -8,7 +8,6 @@
 
 import (
 	"fmt"
-	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strings"
@@ -566,7 +565,7 @@
 	if err != nil {
 		return err
 	}
-	err = ioutil.WriteFile(_filePath(dir, name), data, info.Mode())
+	err = os.WriteFile(_filePath(dir, name), data, info.Mode())
 	if err != nil {
 		return err
 	}
